moped-editor/src/layouts/DashboardLayout/DashboardLayout.js

```diff
@@ -17,7 +17,9 @@ const useStyles = makeStyles(theme => ({
     display: "flex",
     flex: "1 1 auto",
     overflow: "hidden",
-    paddingTop: 64,
+    // If in staging environment, add extra padding
+    // to make room for staging environment info alert
+    paddingTop: process.env.REACT_APP_HASURA_ENV === "staging" ? 114 : 64,
```

Comment:
These magic numbers make me a little uneasy. Also seems like they should have a unit, right? It all looks good in staging currently so not asking for any changes, just pointing it out.

Reply:
thanks for pointing that out, @mateoclarke ! I was wondering if there's a better solution, would be curios to know the best way to handle styling in these different environments!

Reply:
Sometimes, especially with things like padding, magic numbers are unavoidable. The most basic thing to do is something like:

```js
const stagingBannerPadding = "114px"
const standardPadding = "64px"
...
  paddingTop: process.env.REACT_APP_HASURA_ENV === "staging" ? stagingBannerPadding : standardPadding,
...
```
This is a way of making the code and variable assignments serve a similar or complementary role to code comments.

Reply:
that is much cleaner, thank you! i'll do that next time or maybe make these tweaks in my next pull request!